modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,396 @@
+ROSA
+endif::openshift-rosa[]
+clusters. You must set tags separately for these resources.
+====
++
+. Log into the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster as a user with the `cluster-admin` role and create a new project using the following command:
++
+[source, terminal]
+----
+$ oc new-project aws-load-balancer-operator
+----
++
+. Link the AWS IAM role you created previously with the ALBO service account and create a Kubernetes Secret for the ALBO:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+stringData:
+  credentials: |
+    [default]
+    role_arn = <aws-load-balancer-operator-role-arn> <1>
+    web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+----
+<1> Replace `<aws-load-balancer-operator-role-arn>` with the AWS IAM role you created previously.
++
+. Create ALBO:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+spec:
+  upgradeStrategy: Default
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+spec:
+  channel: stable-v1.0
+  installPlanApproval: Automatic
+  name: aws-load-balancer-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: aws-load-balancer-operator.v1.0.0
+----
++
+. Create an AWS ALBC:
++
+[source,yaml]
+----
+apiVersion: networking.olm.openshift.io/v1
+kind: AWSLoadBalancerController
+metadata:
+  name: cluster
+spec:
+  subnetTagging: Manual
+  credentials:
+    name: aws-load-balancer-controller-cluster
+----
++
+[IMPORTANT]
+====
+Because AWS ALBCs do not support creating ALBs associated with both AZs and AWS LZs,
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+clusters can have ALBs associated exclusively with either AWS LZs or AZs but not both simultaneously.
+====
++
+For more information regarding AWS ALBC configurations, see the following topics:
+
+* link:https://docs.openshift.com/container-platform/4.13/networking/aws_load_balancer_operator/multiple-ingress-through-single-alb.html[Creating multiple ingresses]
+* link:https://docs.openshift.com/container-platform/4.13/networking/aws_load_balancer_operator/add-tls-termination.html[Adding TLS termination]
+
+. Confirm successful installation by running the following commands:
+.. Gather information about pods within the project:
++
+[source, terminal]
+----
+$ oc get pods -n aws-load-balancer-operator
+----
+.. View the logs within the project:
++
+[source, terminal]
+----
+$ oc logs -n aws-load-balancer-operator deployment/aws-load-balancer-operator-controller-manager -c manager
+----
+.Verification
```

Comment:
```suggestion

.Verification
```
Need to insert a blank line. Otherwise the heading is indented with the steps (see the preview).

![image](https://github.com/openshift/openshift-docs/assets/86380613/829b4c5c-ebf0-4226-8491-5f7b0a9a1f07)